dev/TeachingTip/TeachingTip.cpp

```diff
@@ -53,6 +54,22 @@ void TeachingTip::OnApplyTemplate()
     m_tailEdgeBorder.set(GetTemplateChildT<winrt::Grid>(s_tailEdgeBorderName, controlProtected));
     m_tailPolygon.set(GetTemplateChildT<winrt::Polygon>(s_tailPolygonName, controlProtected));
 
+    if (winrt::IUIElement10 uiElement10 = *this)
+    {
+        if (auto xamlRoot = uiElement10.XamlRoot())
```

Comment:
>XamlRoot [](start = 40, length = 8)

I'm not sure if the XamlRoot is set up fully by OnApplyTemplate - it may need to wait for Loaded, I'm also not sure what we do if the root changes (like the element moves between windows).  Jesse would know those details for sure.